src/components/darien/DarienHome.jsx

```diff
@@ -17,47 +19,54 @@ import {
-        <Section
-          className="program-home__section"
-          align="center"
-          colorIndex="accent-3"
-          direction="column"
-          margin={{ vertical: 'none', horizontal: 'none' }}
-          pad={{ vertical: 'none', horizontal: 'none' }}
-          justify="center"
-        >
-          <Box align="center" direction="column">
-            <Paragraph>Investigate ecological questions by exploring trail camera data using an interactive map. Filter and download data to perform analyses and test hypotheses. If you are an educator, you can set up private classrooms and invite your students to join. Curate data sets or let your students explore on their own. Guided activities and supporting educational resources are also available. If you are a student or you simply want to explore the data, click the Explorer button.</Paragraph>
-            <Paragraph>Are you an educator or a student/explorer? Make your selection to get started!</Paragraph>
+  return (
+    <ProgramHome className="darien-home">
+      <Hero
+        className="program-home__hero"
+        background={<Image src="https://placeimg.com/1000/1000/nature/any" fit="cover" />}
+        backgroundColorIndex="dark"
+        size="medium"
+      >
+        <Box align="center"><Heading className="program-home__header">{name}</Heading></Box>
+      </Hero>
+      <Section
+        className="program-home__section"
+        align="center"
+        colorIndex="accent-3"
+        direction="column"
+        margin={{ vertical: 'none', horizontal: 'none' }}
+        pad={{ vertical: 'none', horizontal: 'none' }}
+        justify="center"
+      >
+        <Box align="center" direction="column">
+          <Paragraph>Investigate ecological questions by exploring trail camera data using an interactive map. Filter and download data to perform analyses and test hypotheses.</Paragraph>
+          
+          <Box pad="medium" size="medium">
+            <Paragraph>If you are an educator, you can set up private classrooms and invite your students to join. Curate data sets or let your students explore on their own. Guided activities and supporting educational resources are also available. {(signedIn) ? null : '(Sign In required)'}</Paragraph>
+            {(signedIn)
+              ? <Button path="/wildcam-darien-lab/educators/" label="Educator" />
+              : <Button type="button" className="button--secondary" onClick={Actions.auth.toggleOauthModal} label="Sign In" />
+            }            
           </Box>
-          <Box align="center" direction="row" justify="between" pad="medium" size="medium">
-            <Button path="/wildcam-darien-lab/educators/" label="Educator" />
-            <Button path="/wildcam-darien-lab/students/" label="Explorer" />
+          
+          <Box pad="medium" size="medium">
+            <Paragraph> Alternatively, you can simply explore the data.</Paragraph>
+            <Button path="/wildcam-darien-lab/map/" label="Explorer" />
```

Comment:
Maybe add `className="button--secondary"` here so this button correctly gets hover/focus styles? I still have had a moment to figure out why Grommet is being inconsistent with the styles.